Assets/HoloToolkit/UX/Scripts/Receivers/InteractionReceiver.cs

```diff
@@ -70,8 +70,15 @@ public virtual void OnEnable()
         /// </summary>
         public virtual void OnDisable()
         {
-            InputManager.Instance.RemoveGlobalListener(gameObject);
-            FocusManager.Instance.PointerSpecificFocusChanged += OnPointerSpecificFocusChanged;
+            if (InputManager.IsInitialized)
+            {
+                InputManager.Instance.RemoveGlobalListener(gameObject);
+            }
+
+            if (FocusManager.IsInitialized)
+            {
+                FocusManager.Instance.PointerSpecificFocusChanged -= OnPointerSpecificFocusChanged;
```

Comment:
Isn't this an event in the InputManager?

We probably don't need to subscribe to this kind of event if we can get it through the interface.  Might see a perf improvement here.

Reply:
My change was wrapping the lines in the `IsInitialized` checks to prevent null refs on teardown.

That sounds like something to look into though. Perhaps https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/master/Assets/HoloToolkit/Input/Scripts/InputHandlers/IPointerSpecificFocusable.cs

Reply:
Exactly the interface I was talking about :)